connectivity_summary.py

```diff
@@ -34,6 +34,12 @@ def arg_to_date(arg):
                          '' 'for which there are >= sweep_threshold number of sweeps/stimulus set. Two thresholds'
                          '' 'are set one for induction protocols (default=5) and one for recovery (default=10')
 parser.add_argument('files', nargs='*', type=os.path.abspath)
+parser.add_argument('--cre_type', nargs=2, type=str)
```

Comment:
Make it `--cre-type`  (just because flags tend to use '-' rather than '_' by convention)